docs-v2/pages/connect/_meta.tsx

```diff
@@ -9,7 +9,10 @@ export default {
     "title": "Quickstart",
   },
   "workflows": {
-    "title": "Running workflows",
+    "title": "Running Workflows",
```

Comment:
Generally we follow the Google style guide, which [recommends "sentence case" for titles](https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings), so please revert this change

Reply:
Hmm this is the only title that follows that convention 🤔

Reply:
I can clean up the rest in a separate PR